### ISSUE TITLE
Some parts of Out of Wind show up with a black background

### ISSUE THREAD

**Issue by Lord-McSweeney** [ruffle-rs/ruffle#10455]
### Describe the bug

SWF: http://i.notdoppler.com/files/outofwind.swf (https://www.notdoppler.com/outofwind.php)
The backgrounds should be transparent, not black. This is visible at the menu and when clicking play- after the latter, buttons in the top-right are obscured.

### Expected behavior

Lightspark renders them semi-correctly (though it has massive visual issues with the game)- see https://github.com/lightspark/lightspark/issues/1056.

### Affected platform

Browser's extension

### Operating system

Linux

### Browser

Chrome 110

**Comment by n0samu** [comment]
I think I figured out a small piece of this puzzle, at least. Let's focus on just sprite 608, which Ruffle renders incorrectly: [sprite608.zip](https://github.com/ruffle-rs/ruffle/files/11108694/sprite608.zip)
![image](https://user-images.githubusercontent.com/71368227/228754583-719f6dd4-abb5-456d-9e45-e8a5979e8061.png)

The incorrect rendering is actually caused by the windmill sprite placed within it. The sprite consists of a PlaceObject 2 tag and a PlaceObject 3 tag which both place the same character (479). The PlaceObject3 tag uses a ColorTransform and the Overlay blend mode to enhance the colors of the original sprite. But similarly to #9304, the Overlay blend mode is rendered incorrectly by Ruffle. As seen in the attached `sprite608_windmillmod.swf`, if we change the blend mode to Normal, the problem goes away:
![image](https://user-images.githubusercontent.com/71368227/228756655-39fa8d21-7b07-42f8-adc9-0fbc1bedb37a.png)

**Comment by n0samu** [comment]
The wrong rendering of sprite 608 started with commit 8db8e4669adb6bc5498b19af66db78a10ce0cc50 of #9096. Same regression as #9304.

**Comment by n0samu** [comment]
The black background issue on the main menu is actually caused by a function inside the game that rasterizes sprites.
![image](https://github.com/ruffle-rs/ruffle/assets/71368227/2b527b2d-f1b8-41fc-83ce-7e3fb9a6dbf9)
The function is in `menu.page.BasePage`:
```as
public function RasterizeChildren() : void
{
    var _loc1_:Sprite = null;
    var _loc2_:int = 0;
    while(_loc2_ < this.fVisual.numChildren)
    {
        if(this.fVisual.getChildAt(_loc2_) is Sprite)
        {
            _loc1_ = Sprite(this.fVisual.getChildAt(_loc2_));
            if(_loc1_.toString().indexOf("Rastr") > 0)
            {
                SpriteUtils.RasterizeToSprite(_loc1_,_loc1_);
                _loc1_.useHandCursor = false;
                _loc1_.mouseChildren = false;
                _loc1_.mouseEnabled = false;
                _loc1_.hitArea = null;
            }
        }
        _loc2_++;
    }
}
```
The menu graphic sprite (609) is named "pgMainMenu_Island_Rastr" so the RasterizeToSprite function gets called. If I remove the function call, the black background problem disappears:
![image](https://github.com/ruffle-rs/ruffle/assets/71368227/db684e59-1399-4026-83af-6830a028896c)

The game has similar rasterization functions in several places, so I think it's reasonable to assume that all of the black background problems are related to these.

**Comment by n0samu** [comment]
(Re)confirmed that the black background problem also started with the same commit 8db8e46.

**Comment by n0samu** [comment]
Fixed by #11674
